src/state/data-model-list/selectors.js

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from 'reselect';
+
+export const getDataModels = state => state.dataModelList;
+
+export const getListDataModels = createSelector(
+  [getDataModels],
+  dataModelList => dataModelList.list,
```

Comment:
This list should go into the `dataModels` state, and not having its own global key